doc/generate-api-docs.sh

```diff
@@ -49,12 +49,30 @@ LIBS_SORTED=$(
     | egrep '^'$(echo $LIBS|sed 's/ /|/g')'$'
 )
 
+dependencies=''
+
 for LIB in $LIBS_SORTED; do
     cd $LIB
+    version=$(git tag --sort=taggerdate | grep $LIB'_' | tail -n1 | cut -d'_' -f2 | cut -d'v' -f2)
+    dependency=$LIB' = { version = "'$version'"'
     ARGS=""
     if FEATURES="$(cargo read-manifest|jq -r '.metadata.docs.rs.features | join(",")' 2> /dev/null)"; then
         ARGS="--features $FEATURES"
+        dependency=$dependency', features = ["'$FEATURES'"]'
     fi
-    cargo doc --no-deps --lib $ARGS
+    dependency=$dependency' }'
+    dependencies=$dependencies$dependency$'\n'
+    #cargo doc --no-deps --lib $ARGS
     cd -
 done
+
+pushd $WORKSPACE
+cargo new foo
+cd foo
+echo "$dependencies" >> Cargo.toml
+cargo doc
```

Comment:
Doesn't this generate documentation for crate foo as well?

Reply:
It does, but I copy the libs' docs to rust-sgx/target/doc

Reply:
The cargo doc works for:

sgx-isa = { version = "0.2.0" }
rs-libc = { version = "0.1.0" }
fortanix-sgx-abi = { version = "0.3.2", features = ["docs"] }
dcap-ql-sys = { version = "0.1.0" }
sgxs = { version = "0.6.0" }
sgxs-loaders = { version = "0.1.0" }

but fails for 

enclave-runner = { version = "0.1.0" }

and onwards

report-test = { version = "0.1.0" }
dcap-ql = { version = "0.1.0" }
aesm-client = { version = "0.1.0" }

I need to resolve this issue.